tools/mininet/examples/trellis/README.md

```diff
@@ -54,6 +54,7 @@ reference:
 | `make mn-log`       | Show the Mininet log (i.e., the CLI output)            |
 | `make netcfg`       | Push netcfg.json file (network config) to ONOS         |
 | `make reset`        | Reset the tutorial environment                         |
+| `make mn-host`      | Run a command in a Mininet host.                       |
```

Comment:
It would be nice to provide an example of how to use this new command (make target) in the instructions below, for example, to start a ping from one of the two hosts.